clap_derive/Cargo.toml

```diff
@@ -18,6 +18,7 @@ include.workspace = true
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
+rustdoc-args = ["--generate-link-to-definition"]
```

Comment:
This one isn't as needed (I should probably drop the entire table).  There isn't a reason for people to directly browse ths package

Reply:
Reverting this change then.